src/CommonDBChild.php

```diff
@@ -865,11 +865,12 @@ public static function showAddChildButtonForItemForm(
             $lower_name         = strtolower(get_called_class());
             $child_count_js_var = htmlescape('nb' . $lower_name . 's');
             $div_id             = htmlescape("add_" . $lower_name . "_to_" . $item->getType() . "_" . $items_id);
+            $add_label          = sprintf(__s('Add a new %s'), static::getTypeName());
```

Comment:
`static::getTypeName()` must be escaped too.

```suggestion
            $add_label          = htmlescape(sprintf(__('Add a new %s'), static::getTypeName()));
```